trpc/router/filter.ts

```diff
@@ -7,21 +7,23 @@ import { connectionMap } from '../queries/connections';
 import { createFilterSchema } from '../../src/schema/filter';
 import { ToggleSubscriptionSchema } from '../../src/schema/common';
 
+const fitlerIncludeSchema = {
```

Comment:
`includeStargizersAndActivity` or much better to separate it `includeStargizers`, `includeActivity`